cloudfront/wellcomelibrary.org/edge-lambda/src/tests/wellcomeLibraryEncoreRedirect.test.ts

```diff
@@ -0,0 +1,114 @@
+import * as origin from '../wellcomeLibraryEncoreRedirect';
+import testRequest from './testEventRequest';
+import { Context } from 'aws-lambda';
+import { results, resultWithIdentifier } from './catalogueApiFixtures';
+import { expectedRedirect } from './testHelpers';
+import axios from 'axios';
+import { expect, jest, test } from '@jest/globals';
+import { CatalogueResultsList } from '../catalogueApi';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
```

Comment:
Not one for this PR but I'm very impressed by https://mswjs.io/ for mocking HTTP requests, it's definitely better than using mocks like this

Reply:
As you can probably guess, this is copy/pasted from other tests. I'm going to leave it because I suspect this code is going to be frozen once we finish turning off wellcomelibrary.org, but good to know for other projects.